client/a8c-for-agencies/sections/sites/controller.tsx

```diff
@@ -1,18 +1,45 @@
 import { type Callback } from '@automattic/calypso-router';
+import Layout from 'calypso/a8c-for-agencies/components/layout';
+import LayoutBody from 'calypso/a8c-for-agencies/components/layout/body';
+import LayoutHeader, {
+	LayoutHeaderTitle as Title,
+} from 'calypso/a8c-for-agencies/components/layout/header';
+import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
+import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
 import SitesSidebar from '../../components/sidebar-menu/sites';
 
 export const sitesContext: Callback = ( context, next ) => {
-	context.header = <div>Header</div>;
 	context.secondary = <SitesSidebar path={ context.path } />;
-	context.primary = <div>sites</div>;
+	context.primary = (
+		<Layout title="Overview" wide sidebarNavigation={ <MobileSidebarNavigation /> }>
```

Comment:
Are we planning to update all the titles later? It says "Overview" for all the pages.

Reply:
Good catch. this has been fixed 👍